docs/docs/api/04-Relations/or.md

```diff
@@ -0,0 +1,43 @@
+---
+displayed_sidebar: apiSidebar
+---
+# or
+
+Logic or operator
+
+**Examples:**
+```text 
+scene 1
+--------------  --------------
```

Comment:
```suggestion
--------------  ---------------
```

Reply:
To explain: I want the boxes to look neat (Sorry for my inner Sheldon) 😉

Reply:
I am also a Bazinga fan :laughing: